sdks/js/src/types.ts

```diff
@@ -789,7 +790,6 @@ const ConversationVisibilitySchema = FlexibleEnumSchema([
 ]);
 
 const ConversationWithoutContentSchema = z.object({
-  id: ModelIdSchema,
```

Comment:
not type-check failure when you remove this one ?

Reply:
no error , but it was actually an issue with the api call itself which was not returning the id . Fixing it instead of changing the types